CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include (utils.cmake)
 
 disallow_intree_builds()
 
+# API version - be sure to update utf8proc.h and Makefile, too!
```

Comment:
(It's error-prone that we have to manually synchronize 3 files for the API version, and another 3 for the ABI version.  Would be good to have a script that does that.)

Reply:
`make distcheck` at least tests that these are consistent.

Reply:
Currently still inconsistent?